shared/actions/platform-specific/index.desktop.js

```diff
@@ -81,6 +82,10 @@ const writeElectronSettingsNotifySound = (_: any, action: ConfigGen.SetNotifySou
   action.payload.writeFile &&
   SafeElectron.getIpcRenderer().send('setAppState', {notifySound: action.payload.sound})
 
+const writeLastSentXLM = (_: any, action: SetLastSentXLMPayload) =>
```

Comment:
the reason state is passed in first is cause the saga runs after the reducer, hence `state.wallet.lastSentXLM` already has the value in it, so you just can do
```
const writeLastSentXML = state => 
```

Reply:
But we need the writeFile value too, right? to know whether this one is supposed to be persisted?